website/docs/docs/build/metrics.md

```diff
@@ -159,7 +159,27 @@ metrics:
 ### Available properties
 Metrics can have many declared **properties**, which define aspects of your metric. More information on [properties and configs can be found here](https://docs.getdbt.com/reference/configs-and-properties).
 
-<VersionBlock firstVersion="1.3">
+<VersionBlock firstVersion="1.4">
+
+| Field       | Description                                                 | Example                         | Required? |
+|-------------|-------------------------------------------------------------|---------------------------------|-----------|
+| name        | A unique identifier for the metric                          | new_customers                   | yes       |
+| model       | The dbt model that powers this metric                       | dim_customers                   | yes (no for `derived` metrics)|
+| label       | A short for name / label for the metric                     | New Customers                   | yes        |
+| description | Long form, human-readable description for the metric        | The number of customers who.... | no        |
+| calculation_method | The method of calculation (aggregation or derived) that is applied to the expression  | count_distinct | yes       |
+| expression  | The expression to aggregate/calculate over | user_id, cast(user_id as int) | yes       |
+| timestamp   | The time-based component of the metric                      | signup_date                     | no       |
+| time_grains | One or more "grains" at which the metric can be evaluated. For more information, see the "Custom Calendar" section.   | [day, week, month, quarter, year]              | no       |
```

Comment:
@mirnawong1 would that work? This table is already nested in a version block because we've got previous versions for the change to `type` and `sql`. For some reason I've got it in my head that version blocks couldn't be nested within one another.